File: src/kyu6/unspinWords.cs
// CSharp

using System;

public class Kata
{
  public static string SpinWords(string sentence)
  {
    string placeholder = "";
    string[] sentence_ = sentence.Split(' ');
    for(int i = 0; i < sentence_.Length; i++)
    {
      if(sentence_[i].Length >= 5)
      {
        for(int i2 = sentence_[i].Length-1; i2 >= 0; i2--)
        {
          placeholder = placeholder + sentence_[i][i2];
        }
        sentence_[i] = placeholder;
        placeholder = "";
      }
    }
    return String.Join(" ", sentence_);
  }
}

File: src/kyu6/unspinWords.js
// JavaScript
function spinWords(sentence){
  sentence = sentence.split(' ')
  for(i = 0; i < sentence.length; i++)
  {
    if(sentence[i].length >= 5)
    {
      sentence[i] = sentence[i].split('').reverse().join('');
    }
  }
  return sentence.join(' ');
}
